### PR TITLE
Updating to support whizzml v0.21.1

### PR DESCRIPTION
**Pull request by osroca**
As usual, we'll need a new release tag.

### CODE REVIEW

File: WhizzML.sublime-completions [hunk 1]
```diff
@@ -272,6 +277,7 @@
         {"trigger": "wait-forever", "contents": "wait-forever"},
         {"trigger": "wait-forever*", "contents": "wait-forever*"},
         {"trigger": "when", "contents": "when"},
+        {"trigger": "with-time-log\tLog running time", "contents": "with-time-log ($1)"},
```
Comment (jaor) [inline WhizzML.sublime-completions]: i would say "Evaluate body and log evaluation time"

File: WhizzML.sublime-completions [hunk 2]
```diff
@@ -178,6 +180,7 @@
         {"trigger": "matches?", "contents": "matches?"},
         {"trigger": "max\tCompute the maximum of", "contents": "max $1"},
         {"trigger": "max-key", "contents": "max-key"},
+        {"trigger": "md5\tGenerate the Hash (MD5) of a string", "contents": "md5 $1"},
```
Comment (jaor) [inline WhizzML.sublime-completions]: i would say "Compute the MD5 hash of a string"  (and same below with SHA1, SHA256)

File: WhizzML.sublime-completions [hunk 3]
```diff
@@ -133,6 +134,7 @@
         {"trigger": "lambda", "contents": "lambda"},
         {"trigger": "last\tGet last element of a list", "contents": "last"},
         {"trigger": "let", "contents": "let"},
+        {"trigger": "levenshtein\tCompute (non-neg integer) the distance between two string values", "contents": "levenshtein $1 $2"},
```
Comment (jaor) [inline WhizzML.sublime-completions]: nit: i'd put the `(non-neg integer)` at the end, and use non-negative (unless then the string is too long).

Comment (osroca) [inline WhizzML.sublime-completions]: It is already too long but I will change it ...

**Comment by osroca** [comment]
nits fixed.

**Comment by osroca** [comment]
Tagged 0.1.5
Sorry about the auto-merge

**Pull request merged**